### PR TITLE
Dev/update to vtk8

### PR DESCRIPTION
**Pull request by RustyBlue**
Updates vivia to build against vtk 8, and address all the (obvious?) depracation issues.

### CODE REVIEW

File: Libraries/VgVideo/vgKwaVideoClip.cxx [hunk 1]
```diff
@@ -24,7 +24,7 @@
 #include <vsl/vsl_binary_io.h>
 #include <vsl/vsl_stream.h>
 #include <vsl/vsl_vector_io.h>
-#include <vsl/vsl_vector_io.txx>
+#include <vsl/vsl_vector_io.hxx>
```
Comment (mwoehlke-kitware) [inline Libraries/VgVideo/vgKwaVideoClip.cxx]: Could you please make these changes in a separate PR?

File: Libraries/VtkVgSceneGraph/vtkVgGeode.cxx [hunk 2]
```diff
@@ -145,15 +145,24 @@ void vtkVgGeode::Update(vtkVgNodeVisitorBase& nodeVisitor)
...
       // \TODO: We are dealing with only 3d props. We need to find a way for
       // setting a user transform to vtkProp if needed.
-      vtkProp3D* currentProp = static_cast<vtkProp3D*>(this->ActiveDrawables->GetNextProp());
-      if (currentProp)
+      vtkProp3D* currentProp =
+        static_cast<vtkProp3D*>(this->ActiveDrawables->GetNextProp());
+      // Divide through by 3,3 component to be assured the 3,3 component is 1
+      // (and more importantly, greater than 0)
+      double w = this->FinalMatrix->GetElement(3, 3);
+      if (currentProp && w != 0)
```
Comment (mwoehlke-kitware) [inline Libraries/VtkVgSceneGraph/vtkVgGeode.cxx]: This causes the later declaration to shadow. Consider inlining `w` here instead?

File: Libraries/VgVideo/vgKwaVideoClip.cxx (hunk 1 above)
Comment (RustyBlue) [reply Libraries/VgVideo/vgKwaVideoClip.cxx]: Definitely - didn't intend for this to slip in to this branch.  As we talked about, I don't want to get pulled in to vxl issues (this should be a problem on Linux as well as Windows, so my choice of vxl may be bad).

File: Libraries/VtkVgSceneGraph/vtkVgGeode.cxx (hunk 2 above)
Comment (RustyBlue) [reply Libraries/VtkVgSceneGraph/vtkVgGeode.cxx]: These changes surprised me... I want to blame it on git.

**Comment by RustyBlue** [comment]
@mwoehlke-kitware - Take another look.  Should hopefully be good-to-go.
